run_sg_tests.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# Exit on failure
+set -e
+
+# In case the docker image is not built with the latest testkit checkout
+git pull
+
+# Generate cluster configs
+python libraries/utilities/generate_clusters_from_pool.py
+
+# Run single test
+pytest -s --pdb --mode=cc --xattrs --server-version=5.0.0-2873 --sync-gateway-version=1.5.0-375 -k "test_purge" testsuites/syncgateway/functional/tests
```

Comment:
The file says run_sg_tests but it is only running test_purge. 
Also, I'd make the server-version and sync-gateway-version as command line args.